lib/manageiq/appliance_console/external_httpd_authentication/external_httpd_configuration.rb

```diff
@@ -243,6 +248,14 @@ def self.fetch_sssd_domain
       config_file_read(SSSD_CONFIG)[/\[domain\/(.*)\]/, 1] if File.exist?(SSSD_CONFIG)
     end
 
+    def self.fetch_saml_configured?
```

Comment:
This method naming is strange. A method with a question mark usually indicates `true` or `false` return value but this method is returning a status string. I think this might be better if it was called `saml_status`

Same applies to the `fetch_oidc_configured?` method

Reply:
Good point. My initial approach was to have these methods return `true` or `false`. I forgot to rename them when I decided on this approach.

Reply:
Thank you @carbonin